src/should-forward-property.js

```diff
@@ -11,28 +11,40 @@ import reactHTMLAttributes from 'react-html-attributes'
 import reactProps from './react-props'
 
 const globalReactHtmlProps = reactHTMLAttributes['*']
+const supportedSVGTagNames = reactHTMLAttributes.elements.svg
 
 // these are valid attributes that have the
 // same name as CSS properties, and is used
 // for css overrides API
 const cssProps = ['color', 'height', 'width']
 
 /* From DOMProperty */
-// eslint-disable-next-line max-len
-const ATTRIBUTE_NAME_START_CHAR = ':A-Z_a-z\\u00C0-\\u00D6\\u00D8-\\u00F6\\u00F8-\\u02FF\\u0370-\\u037D\\u037F-\\u1FFF\\u200C-\\u200D\\u2070-\\u218F\\u2C00-\\u2FEF\\u3001-\\uD7FF\\uF900-\\uFDCF\\uFDF0-\\uFFFD'
+const ATTRIBUTE_NAME_START_CHAR =
+  // eslint-disable-next-line max-len
+  ':A-Z_a-z\\u00C0-\\u00D6\\u00D8-\\u00F6\\u00F8-\\u02FF\\u0370-\\u037D\\u037F-\\u1FFF\\u200C-\\u200D\\u2070-\\u218F\\u2C00-\\u2FEF\\u3001-\\uD7FF\\uF900-\\uFDCF\\uFDF0-\\uFFFD'
 // eslint-disable-next-line max-len
 const ATTRIBUTE_NAME_CHAR = `${ATTRIBUTE_NAME_START_CHAR}\\-.0-9\\u00B7\\u0300-\\u036F\\u203F-\\u2040`
 const isCustomAttribute = RegExp.prototype.test.bind(
   new RegExp(`^(data|aria)-[${ATTRIBUTE_NAME_CHAR}]*$`),
 )
 
+const isSvgTag = tagName => supportedSVGTagNames.indexOf(tagName) !== -1
 const isHtmlProp = (name, tagName) => {
-  const elementAttributes = reactHTMLAttributes[tagName]
+  let elementAttributes
+
+  if (isSvgTag(tagName)) {
+    // all SVG attributes supported by React are grouped under 'svg'
+    elementAttributes = reactHTMLAttributes.svg
+  } else {
+    elementAttributes = reactHTMLAttributes[tagName]
+  }
 
-  return globalReactHtmlProps.indexOf(name) !== -1 ||
+  return (
+    globalReactHtmlProps.indexOf(name) !== -1 ||
     (elementAttributes === undefined ?
```

Comment:
Maybe instead of this ternary, we could initialize `elementAttributes` to an empty array.